timemachine/fe/topology.py

```diff
@@ -318,14 +318,14 @@ def parameterize_nonbonded_pairlist(
         inclusion_idxs, rescale_mask = [], []
         for i in range(self.mol.GetNumAtoms()):
             for j in range(i + 1, self.mol.GetNumAtoms()):
-                scale_factor = exclusions_kv.get((i, j), (0.0, 0.0))  # how much to remove
-                rescale_factor = 1 - np.array(scale_factor)  # how much to keep
+                scale_factor = exclusions_kv.get((i, j), np.zeros(2))  # how much to remove
```

Comment:
Possible to specify a `dtype=np.float64` here? (To avoid guessing if it's `int32` vs `float64` vs `float32`)